.github/workflows/format.yml

```diff
@@ -55,12 +53,16 @@ jobs:
       - uses: actions/checkout@v4
 
       - name: Run ruff format check for ${{ matrix.files.path }}
+        id: format-check
         uses: astral-sh/ruff-action@v1
+        # Allow the job run to pass when this step fails
+        continue-on-error: true
         with:
           args: "format --check --diff"
           src: "${{ matrix.files.path }}"
+          version: 0.8.1
 
       - name: Annotate unformatted file
-        if: ${{ failure() }}
+        if: ${{ steps.format-check.outcome }} == 'failure'
         run: |
-          echo "::warning file=${{ matrix.files.path }},title=File format check failed::Run 'ruff format ${{ matrix.files.path }}'"
+          echo "::notice file=${{ matrix.files.path }},title=Unformatted file::Consider running 'ruff format ${{ matrix.files.path }}'"
```

Comment:
Please tell the user that the formatted code can be found in the check logs.

(technically: the diff but no need to mention that here)